notebooks/Middle_ear_Inter_subject_analysis.ipynb

```diff
@@ -43,45 +43,37 @@
     "import scipy\n",
```

Comment:
<p>Unecessary definition of parameters : EDF_files, the name of result file hould be put in the cell where it is used for more covenience</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lkorczowski/Tinnitus-n-Sleep/pull/85/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
done